bait-files/src/main/webapp/userHistory.js

```diff
@@ -0,0 +1,52 @@
+const url =
+    'https://step-bait-project-2020.uc.r.appspot.com/review.html?reviewId=';
+
+function getCompletedReviews() {
+  fetch('/userHistory?is-pending-review=false')
+      .then(response => response.json())
+      .then((reviews) => {
```

Comment:
Styling: be consistent with your decision to use parentheses or not for single arguments to an arrow function.  Here you've used parentheses for "(reviews)" but no parentheses for "response".  Either can be fine, but it's best to do the same thing in every case =)

Reply:
Removed